### DISCUSSION TITLE
Downloading Tomcat jars

### DISCUSSION THREAD

**Discussion by drumichael611** [redisson/redisson#5102] [Q&A] [answered]
I am attempting to download the .jar files needed to get Tomcat sessions into Redis with Redisson, however when I click the link in the doc (https://redisson.org/articles/redis-based-tomcat-session-management.html) to download, I am asked for a login for sonatype, with no indication on how to create an account.  Any ideas? Thanks!

**Comment by mrniko** [comment, marked as answer]
Links were fixed. Thanks for report
